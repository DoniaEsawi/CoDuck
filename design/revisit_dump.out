------------ -------------
Identifier   Revisit Type
------------ -------------
add          Parameter Check
a            Parameter Check
b            Parameter Check
