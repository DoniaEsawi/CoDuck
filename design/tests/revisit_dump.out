------------ -------------
Identifier   Revisit Type
------------ -------------
func1        Parameter Check
print        Parameter Check
add          Parameter Check
func2        Parameter Check
